azuracast/liquidsoap_custom.liq

```diff
@@ -445,7 +449,8 @@ radio = switch(track_sensitive=false, transitions=[
     ({6w and 17h-18h59m}, once(playlist_rnr666)),
     ({6w and 19h-20h59m}, once(playlist_mood_sequence)),
     ({6w and 21h00m-21h59m}, once(playlist_brains_sublimating)),
-    ({6w and 10h30m-21h30m}, off_air_ambient_mix),                     #ambient: Merites - Brains
+    ({6w23h30m-7w0h29m}, once(playlist_linear_systems_with_gestalt)),
+    ({6w10h30m-7w0h30m}, off_air_ambient_mix),                     #ambient: Merites - Gestalt
```

Comment:
The logic so far was that the OFF AIR ambient playlist is scheduled until approx. **half** of the last show. The rationale is that if the show ends earlier, then the normal OFF AIR playlist will play.